src/ParseServer.js

```diff
@@ -157,10 +157,10 @@ class ParseServer {
       throw 'When using an explicit database adapter, you must also use an explicit filesAdapter.';
     }
 
-    userSensitiveFields = Array.from(new Set(userSensitiveFields.concat(
-      defaults.userSensitiveFields,
-      userSensitiveFields
-    )));
+    // allow 'userSensitiveFields' from constructor to override default list
+    if (typeof userSensitiveFields === 'undefined'){
+      userSensitiveFields = defaults.userSensitiveFields
+    }
```

Comment:
I think this new behavior makes sense, if you specify 'userSensitiveFields' then that takes precedence over the default list of fields instead of merging the two.

Technically this could be considered a breaking change because users that are currently specifying 'userSensitiveFields' are currently getting 'email' appended to that list so this change would alter that behavior.  That said, the new behavior would just expose an extra field, which shouldn't break anything.  If we really want to avoid making this in a breaking fashion, I can add a new field to the constructor, something like 'includeDefaultUserSensitiveFields' to trigger the new behavior but that seemed a but overkill.